src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -54,19 +60,40 @@ export const TopBarEquipmentSearchDialog: FunctionComponent<
     const enableSearchDialog = useCallback(() => {
         setIsDialogSearchOpen(true);
     }, [setIsDialogSearchOpen]);
+    const { snackWarning } = useSnackMessage();
 
     const onSelectionChange = useCallback(
         (element: EquipmentInfos) => {
             setIsDialogSearchOpen(false);
             updateSearchTerm('');
             addToLocalStorageSearchEquipmentHistory(studyUuid, element);
-            showVoltageLevelDiagram(element);
+            fetchNetworkElementInfos(
+                studyUuid,
+                currentNode?.id,
+                element.type,
+                EQUIPMENT_INFOS_TYPES.MAP.type,
+                element.id,
+                false
+            )
+                .then(() => {
+                    showVoltageLevelDiagram(element);
+                })
+                .catch(() => {
+                    excludeElementFromCurrentSearchHistory(studyUuid, element);
+                    updateSearchTerm('');
```

Comment:
we are already emptying search term above, remove this

Reply:
as discussed we will keep it to refresh list of current history list